tb_plugin/torch_tb_profiler/profiler/module_parser.py

```diff
@@ -296,12 +297,13 @@ def aggregate(key_to_agg, key, op):
         def parse_kernels(kernel_list):
             name_op_to_agg = {}
             for kernel in kernel_list:
-                key = kernel.name + "###" + kernel.op_node.name
+                op_name = "<None>" if kernel.op_node is None else kernel.op_node.name
```

Comment:
provide screen shot for this case.

Reply:
`<None>` or `N/A`? or something else looks pretty?

Reply:
Thanks. N/A may be better.

Reply:
![image](https://user-images.githubusercontent.com/10429114/109294603-5f8dea00-7868-11eb-88b1-6e3ebad03207.png)